### DISCUSSION TITLE
Exam rank 05 - No tests for canonical form

### DISCUSSION THREAD

**Discussion by Coluyanson** [JCluzet/42_EXAM#129] [General]
I wanted to ask you if it would be possible to add tests on canonical form of the 01 and 02 module in the exam 05.
I tried the exam on your project and passed it without implementing the copy constructor and the copy assignment operator overload.
On the exam, however, I failed for not implementing them.

**Comment by bakyt92** [comment]
> I wanted to ask you if it would be possible to add tests on canonical form of the 01 and 02 module in the exam 05.
> 
> I tried the exam on your project and passed it without implementing the copy constructor and the copy assignment operator overload.
> 
> On the exam, however, I failed for not implementing them.

Not passed yet?
